src/python/pants/backend/javascript/subsystems/nodejs.py

```diff
@@ -95,6 +106,39 @@ async def download_known_version(
         download_file = DownloadFile(url, FileDigest(known_version.sha256, known_version.filesize))
         return await Get(DownloadedExternalTool, ExternalToolRequest(download_file, exe))
 
+    class EnvironmentAware(Subsystem.EnvironmentAware):
+        env_vars_used_by_options = ("PATH",)
+
+        search_path = StrListOption(
+            default=["<PATH>"],
+            help=softwrap(
+                f"""
+                A list of paths to search for Node.js distributions.
+
+                This option take precedence over the templated url download,
+                if a version matching the configured version range is found
+                in these paths.
```

Comment:
This makes it less hermetic(I think?), but also what I think is most intuitive to users? I'm guessing here.

Reply:
> And the advantage to doing this is that it avoids the download?

Indeed. That, and a little bit of consistency across backend, what with `go` and `python` backends both defaulting to search_paths to find their runtimes.

Reply:
> That, and a little bit of consistency across backend, what with `go` and `python` backends both defaulting to search_paths to find their runtimes.

While true, that has caused us no end of issues, heh. Hermetic by default would be quite nice: it's what we have for the `java`/`scala` backends.

Reply:
Flipping the switch is simple, so I'll do it!